<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPan</class>
 <widget class="QDialog" name="UserPan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订单管理中心</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1261</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 219, 255);</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>101</y>
      <width>921</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 255, 244);</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>付款</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打印通知单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打印账单</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请在下面勾选项目，然后点击上面的操作按钮进行所需操作：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>操作</string>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>1</y>
     <width>1281</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
